prettyprinter-ansi-terminal/src/Data/Text/Prettyprint/Doc/Render/Terminal/Internal.hs

```diff
@@ -123,15 +123,15 @@ renderLazy sdoc = runST (do
     let push x = modifySTRef' styleStackRef (x :)
         unsafePeek = readSTRef styleStackRef >>= \tok -> case tok of
             []  -> panicPeekedEmpty
-            x:_ -> pure x
+            x:_ -> return x
```

Comment:
I’d rather have a conditional import of Applicative instead of using `return`, in order to keep compatibility hacks in the imports rather than in the code.

Reply:
http://hackage.haskell.org/package/base-4.3.0.0/docs/Control-Monad-ST.html not `Applicative` in GHC-7.0 base (`base-orphans` probably fixes that though)